### PR TITLE
README: better explain relationship of repo/list

### PR DESCRIPTION
**Pull request by davelab6**
Please don't merge yet, will edit after there's more clarity on this

### CODE REVIEW

**Comment by stuaxo** [comment]
In my last contract we used "WIP" in the ticket description to say "don't merge this" having a "work in progress" label could work here.

**Comment by alerque** [comment]
@stuaxo Draft PR's are a relatively new tool in the GitHub toolbox. There is no need for WIP titles, descriptions, or labels. Draft PR's don't have a merge button and they cannot be merged until somebody with the right powers (the same powers that could edit the title or change the labels) marks them as ready for review.

**Comment by stuaxo** [comment]
Fantastic :) tbh I'm having trouble keeping up with all the new things being added to github.

**Comment by davelab6** [comment]
@vlevantovsky I'm sure I got some details wrong but I'd be grateful if you could take a look :)

**Comment by davelab6** [comment]
I've enabled edits by maintainers so you can correct the file on my fork's branch directly.

File: README.md
Comment (stuaxo) [inline README.md]: Should this be -

> reposted **by** older members

?

**Review by alerque — APPROVED** [review]

File: README.md
Comment (alerque) [inline README.md]:
```suggestion
(Sadly file attachments are not available in the archive, but all formal input and output documents can be found in MPEG document registry and on ISO Livelink document repositories (please note that the access to those document archives is password protected and available only to active members of SC29.))
```

**Review by alerque — CHANGES_REQUESTED** [review]

File: README.md
Comment (stuaxo) [reply README.md]: in *the* MPEG document registry

**Comment by davelab6** [comment]
Thanks for the corrections Vlad! I'd like to take one more pass on this before you merge, by EOW.

**Comment by davelab6** [comment]
Note to self, John Hudson posted some stuff on the list relevant to this that I want to pull in

Also useful details in https://lists.aau.at/pipermail/mpeg-otspec/2016-February/000674.html

Also perhaps worth moving to a `/CONTRIBUTING.md` file

Also note proposal format recommendations at https://github.com/MPEGGroup/OpenFontFormat/issues/17#issuecomment-699532588

**Comment by simoncozens** [comment]
This is brilliant, and adds a lot more clarity to expectations around how changes to the standard (ought to) happen.

**Review by alerque — APPROVED** [review]

**Comment by davelab6** [comment]
@vlevantovsky would you like any more changes before merging this? In happy to see it go up and I'll follow up with a second PR with other details later? :)

**Comment by vlevantovsky** [comment]
> If no objections were raised, or if all objections were retraced, ...

Did you really mean to say "retraced", or "retracted"?

**Comment by davelab6** [comment]
Typo

**Review by vlevantovsky — APPROVED** [review]

**Pull request merged**
